src/index.js

```diff
@@ -390,14 +390,31 @@ export default class Map extends Component {
 
   handleTouchEnd = (event) => {
     if (this._touchStartCoords) {
-      event.preventDefault()
+      
       const { center, zoom } = this.sendDeltaChange()
 
       if (event.touches.length === 0) {
+        
+        // if the click started and ended at about
+        // the same place we can view it as a click
+        // and not prevent default behavior.
+        const start = this._touchStartCoords[0]
+        const touch = event.changedTouches[0]
+        const end = [touch.clientX, touch.clientY]
+
+        if (
+          Math.abs(start[0] - end[0]) > 5
+          || Math.abs(start[1] - end[1]) > 5
```

Comment:
could you put this `5` into a constant like the other ones at the top of the file?